lib/carrierwave/base64/adapter.rb

```diff
@@ -17,9 +17,16 @@ module Adapter
       #   mount_base64_uploader :image, ImageUploader,
       #     file_name: -> (u) { u.username }
       #
+      # @example Mount the uploader with a block, overriding the store_dir
+      #   mount_base64_uploader :image, ImageUploader do
+      #     def store_dir
+      #       'images'
+      #     end
+      #   end
+      #
       # @return [Symbol] the defined writer method name
-      def mount_base64_uploader(attribute, uploader_class, options = {})
-        mount_uploader attribute, uploader_class, options
+      def mount_base64_uploader(attribute, uploader_class, options = {}, &block)
+        mount_uploader attribute, uploader_class, options, &block
```

Comment:
do you know, starting from which version `carrierwave` supports this? We should probably update our `gemspec`

Reply:
I checked the code on GitHub, and it looks like the code to pass the block to the Uploader class has been in there since the early days.
https://github.com/carrierwaveuploader/carrierwave/blob/v2.1.0/lib/carrierwave/mount.rb#L134
https://github.com/carrierwaveuploader/carrierwave/blob/v0.8.0/lib/carrierwave/mount.rb#L141
https://github.com/carrierwaveuploader/carrierwave/blob/v0.1.0/lib/carrierwave/mount.rb#L66